src/_data/side-nav.yml

```diff
@@ -12,6 +12,8 @@
           permalink: /codelabs
         - title: Cheatsheet codelab
           permalink: /codelabs/dart-cheatsheet
+        - title: Asynchronous Programming with Dart
```

Comment:
I like the second option since it avoids repetition of "Codelab" in sub-menu items that are members of the "Codelab" menu item, will make this change. I'm assuming I should change the title of the cheatsheet as well -- if not, let me know!

Reply:
On second glance, it doesn't look like we are currently matching codelab titles to their menu item counterparts, so I'll leave the codelab titles alone.